src/rgw/jwt-cpp/jwt.h

```diff
@@ -0,0 +1,1605 @@
+#pragma once
+#define PICOJSON_USE_INT64
+#include "../picojson/picojson.h"
```

Comment:
I have modified the path here to a relative path for including picojson.h. Is this Ok?

Reply:
yes